### DISCUSSION TITLE
Generating type URL instead of any

### DISCUSSION THREAD

**Discussion by johannesbraeunig** [dotansimha/graphql-code-generator#4780] [General] [answered]
Hi,

I just figured out that if we have a field of type `URL` in the gql schema the generator output is always `any`.

```gql
type Something {
  id: ID!
  url: URL!
}
```

```ts
export type Seomthing = {
  id: string;
  url: any;
};
```

How it should be!?

```ts
export type Seomthing = {
  id: string;
  url: URL;
};
```

Is there a way to type `url` with `URL`? Do we miss any configuration? 🤔

**Comment by ardatan** [comment, marked as answer]
You can use `scalars` option like below;
```
config:
  scalars:
     URL: URL
```

**Reply by johannesbraeunig** [reply]
Oh yeah, thank you very much 🙏
